src/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ load("//tools:defaults.bzl", "ts_library")
 exports_files([
   "bazel-tsconfig-build.json",
   "bazel-tsconfig-test.json",
+  "README.md",
```

Comment:
Not for this PR, but we should create separate npm README files for each of our packages

Reply:
Agreed. Should these be pretty much similar to the current one but just have the "correct" package name title and refer to the proper URL for the sources?

Reply:
I created an issue and assigned it to myself for this.